content/api/commands/screenshot.md

```diff
@@ -167,14 +167,15 @@ cy.screenshot('my-screenshot', {
 Screenshot naming follows these rules:
 
 - By default, a screenshot is saved to a file with a path relative to the
-  [screenshots folder](/guides/references/configuration#Folders-Files), appended
```

Comment:
Because the header in the other file was changed, this link also needs to be updated.